### PR TITLE
docs: add CONTRIBUTING documentation for releasing specifically

### LINKED ISSUE #140: Add contributing guidelines
### Subject of the feature

This project has the MDX defaults for support, contributing, issue/pr templates, etc, but some project-specific ways to contribute (e.g., how to publish) are unclear to me.

### Problem

I’m unsure what the procedure is to publish a release.

### Expected behavior

Documentation on which scripts to run, what to do, etc.

### PR DESCRIPTION
**Pull request by JounQin**
close #140

### CODE REVIEW

**Comment by JounQin** [comment]
@wooorm How do you think about this documentation for contributing?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## Core Guidelines
+
+[Read the guidelines on the MDX website][contributing]
+
+## Packages Releasing Difference
+
+This project is a [lerna][] monorepo, so packages releasing is controlled by [lerna][].
+
+1. Make sure you have both GitHub repository and npm write permissions at the same time.
+2. Export an environment named `GH_TOKEN` for `lerna publish`.
```
Comment (wooorm) [inline CONTRIBUTING.md]: People shouldn’t directly use `lerna publish`, right? If it shouldn’t be used, I suggest removing this bullet, as the next one already explains how to pass a GH token?

Comment (JounQin) [reply CONTRIBUTING.md]: @wooorm I meant to talk about exporting the environment globally inside something like `.bash_profile`.

Comment (wooorm) [reply CONTRIBUTING.md]:
```suggestion
2. You need a GitHub token with a `public_repo` scope as `GH_TOKEN` in the
   environment to publish
```

Maybe like this?

**Comment by wooorm** [comment]
This looks great, thanks!

**Pull request merged**